static/js/plot.js

```diff
@@ -233,15 +233,39 @@ function drawTimeSeries(seriesArray, valueKey, selector) {
     );
   }
 
+  // The y-label is based on units, mDenoms, and scalingFactors
+  // seen so far.
+  // As more data is added to Data Commons, this logic may need
+  // further fine-tuning.
   let yLabelText = "";
   if (sample["unit"]) {
     yLabelText = sample["unit"];
   }
-  if (sample["scalingFactor"] == 100) {
-    yLabelText = "% of " + sample["measurementDenominator"];
-  } else if (sample["scalingFactor"]) {
-    yLabelText = "Proportion of " + sample["measurementDenominator"];
+  if (sample["measurementDenominator"]) {
```

Comment:
would be nice have a comment of some sample input and what the label should be? Later can move them to unittest

Reply:
Done